src/Control/Control.js

```diff
@@ -1516,7 +1516,7 @@ var Control = module.exports = kind(
 		var was = this.content;
 		this.content = content;
 
-		if (was != content) this.notify('content', was, content);
+		if (was !== content) this.notify('content', was, content);
```

Comment:
(notimpressed)

Hm, so the alternative, in this situation, would be to explicitly convert `ProgressBar` popup content to strings? That seems a bit messy to me, but I can see the basis for the argument - `content` is specifically defined to be a string. Thoughts?